lib/std/x.zig

```diff
@@ -8,6 +8,7 @@ const std = @import("std.zig");
 
 pub const os = struct {
     pub const Socket = @import("x/os/socket.zig").Socket;
+    pub usingnamespace @import("x/os/io.zig");
     pub usingnamespace @import("x/os/net.zig");
```

Comment:
can you justify the use of `usingnamespace` here? is it really necessary?

Reply:
Nope, no justification - I'm hoping to address this in the next PR as I'll be breaking down the current code in x/net/io.zig further into separate files and removing that line (i.e. changing it to `pub const Reactor = @import("x/os/reactor.zig")`). Just for this PR, I'm exposing the contents of x/net/io.zig using `usingnamespace`.